### PR TITLE
Feature/link to etherscan

### PR DESCRIPTION
**Pull request by tomfelix**

### CODE REVIEW

File: src/requests/Request.tsx [hunk 1]
```diff
@@ -88,14 +91,32 @@ class Request extends PureComponent<RequestStatus> {
...
             }
           </RequestContent>
         </RequestTableCell>
+        <RequestTableCell>
+          <RequestLabel>{Labels.value}</RequestLabel>
+          <RequestContent>
+            {
+              hash ?
+                <LinkWrapper
+                  href={`https://ropsten.etherscan.io/tx/${hash}`}
```
Comment (lukaszblasz) [inline src/requests/Request.tsx]: Put urls in const/configs not inside views.

Comment (lukaszblasz) [reply src/requests/Request.tsx]: Is it always be a ropsten? I think this shluld be taken from ENV variable. You can use ropsten or other provider.

Comment (tomfelix) [reply src/requests/Request.tsx]: we can use only ropsten.

File: src/requests/components/RequestTableHeadCell.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const RequestTableHeadCell = styled.th`
     box-sizing: border-box;
     text-align: center;
     padding: 10px;
-    width: 20%;
+    width: calc(100%/6);
```
Comment (przemyslawjanpietrzak) [inline src/requests/components/RequestTableHeadCell.tsx]: 16.7%

File: src/requests/Request.tsx (hunk 1 above)
Comment (tomfelix) [reply src/requests/Request.tsx]: done

File: src/requests/components/RequestTableHeadCell.tsx (hunk 2 above)
Comment (tomfelix) [reply src/requests/components/RequestTableHeadCell.tsx]: fixed

**Pull request merged**